webserver/views/datasets.py

```diff
@@ -103,6 +113,15 @@ def eval_info(dataset_id):
         author=db.user.get(ds["author"]),
     )
 
+@datasets_bp.route("/<uuid:dataset_id>/evaluation/<uuid:job_id>")
```

Comment:
Can we use the existing view and just make the job id optional? (http://stackoverflow.com/a/16569475). This prevents us from having to maintain the same code in two different places.